bin/fin

```diff
@@ -3097,13 +3105,13 @@ config_generate ()
 	if [[ -f ${yml_file} ]]; then
 		if [[ -d .docksal ]]; then
 			echo-yellow ".docksal directory already exists."
-			_confirm "Remove it and start from scratch?"
-			rm -rf ./.docksal
+			_confirm "Overwrite?"
```

Comment:
It would probably be better to have `fin config generate` and `fin config update`.
`generate` starts from scratch, while update writes over existing stuff without removing the entire `.docksal` folder. Presumably the only reason we'd have this is to preserve the local yml and env files a user may have already locally.